### PR TITLE
bpo-41090: Add support for building "Universal 2" binaries on macOS 11.

### PR DESCRIPTION
**Pull request by ronaldoussoren**
Add a new option to "--with-universal-archs": "universal2". This will build binaries supporting the arm64 and x86_64 architectures, in preparation for "Apple Silicon" systems.

https://bugs.python.org/issue41090

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -16806,7 +16811,7 @@ do
 done
 
 
-SRCDIRS="Parser Parser/pegen Objects Python Modules Modules/_io Programs"
+SRCDIRS="Parser Objects Python Modules Modules/_io Programs"
```
Comment (merwok) [inline configure:16814]: Is this change on purpose or bad merge?

Comment (ned-deily) [reply configure:16814]: A previous commit from another project left `configure` out of sync with ``configure.ac``.  While this corrects that, I'm pushing a separate PR (#21152 )  to fix it right away.

**Comment by ned-deily** [comment]
This PR and its issue have been superseded by GH-22855 and bpo-41100.
